core/src/test/resources/newlines/alwaysBeforeTopLevelStatements.stat

```diff
@@ -144,4 +158,117 @@ object A {
 
   @Singleton
   class B @Inject()(val x: Int)
+
+  @js.native
+  class C() {
+    def c = 5
+  }
+}
+<<< Double annotation
+object a {
+  @annot
+  @bannot
+  class A
+}
+>>>
+object a {
+
```

Comment:
@Daxten Before this PR, iirc, this newline would not have been inserted, right? If so, I'm happy to follow up with another PR changing it back to the old behavior.